2025/02/15/post-mortem.md

```diff
@@ -0,0 +1,194 @@
+  - **Full outage:** Approximately 56 hours until restoration of map edits.
+  - **Total time until HE.net resolved:** About 68 hours.
+
+**Impact:**
+- The primary OpenStreetMap.org website and API were initially completely offline.
+- A read-only backup instance was made available on 15 December 2024 at 11:31 to provide limited functionality.
+- Full functionality was restored on 17 December 2024 at 12:21 by switching to Equinix Internet in Amsterdam.
+- HE.net service was fully restored on 18 December 2024 at 00:29, at which point all usual routes and services returned to normal.
+
+**Root Cause:**
+- An unanticipated failure of HE.net routing equipment in Amsterdam led to a network blackout. The routing hardware had to be replaced, and this process involved them shipping new equipment from California. During this time, OpenStreetMap.org was exposed to their single point of failure with no direct failover ISP in place.
+
+---
+
+## 2. Timeline of Events
+
+Below is a timeline of the incident:
+
+1. **15 December 2024, 03:53 (UTC) – Outage Begins**
+   - HE.net suffers a routing equipment failure in Amsterdam, causing OpenStreetMap.org to become unreachable.
+
+2. **15 December 2024, 04:01 – Initial Detection**
+   - StatusCake monitoring tools detect the outage and send SMS alerts.
+   - Incident is immediately reported to the Operations team.
+
+3. **15 December 2024, 04:03 – Internal Coordination**
+   - Operations team (via IRC) confirms that OpenStreetMap is offline.
+   - By 04:06, Grant confirms that this is an external ISP issue with HE.net.
+
+4. **15 December 2024, 04:18 – First Email to HE.net NOC**
+   - OSM Operations team emails HE.net’s Network Operations Centre (NOC), inquiring if there is any unplanned maintenance.
+
+5. **15 December 2024, 04:24 – HE.net NOC Response**
+   - HE.net confirms via email that there is no scheduled maintenance. They are investigating the outage.
+
+6. **15 December 2024, 04:43 – HE.net Phone Status**
+   - OSM Operations contacts HE.net by telephone.
+   - HE.net states the outage is in Amsterdam, with no estimated time to recover. They are waiting for Equinix "remote hands" to investigate.
+
+7. **15 December 2024, 07:24 – Second Request for ETA**
+   - OSM Operations again requests an estimated time of recovery from HE.net.
+   - No precise estimate provided.
+
+8. **15 December 2024, 11:31 – Read-Only Backup Online**
+   - A read-only backup instance of OpenStreetMap.org (hosted in Dublin) is activated, providing partial functionality.
+   - A maintenance notice is displayed on OpenStreetMap.org, indicating that data edits cannot be saved.
+
+9. **15 December 2024, 11:12 & Ongoing – Additional Enquiries**
+   - OSM Operations team continues to request updates from HE.net at regular intervals, but no ETA is provided.
+
+10. **16 December 2024, TBC - Operations Team Prepare Emergency Full Failover Procedure Documention**
```

Comment:
It was about 10am when I wrote my plan - last edit was 10:11 according to hackmd.

Reply:
Thanks. Fixed.